contrib/gunicorn.conf

```diff
@@ -0,0 +1,22 @@
+# Bind is the ip and port that the Netbox WSGI should bind to
+#
+bind='127.0.0.1:8001'
+
+# Workers is the number of workers that GUnicorn should spawn.
+# Workers should be: cores * 2 + 1.  So if you have 8 cores, it would be 17.
+#
+workers=3
+
+# Threads
+#     The number of threads for handling requests
+#
+threads=3
+
+# Timeout is the timeout between gunicorn receiving a request and returning a response (or failing with a 500 error)
+#
+timeout=120
+
+# ErrorLog
+#     ErrorLog is the logfile for the ErrorLog
+#
+errorlog='/usr/local/netbox/netbox.log'
```

Comment:
`/usr/local/netbox` used as errorlog directory, but rest of documentation uses `/opt/netbox`

Reply:
I can't win...  Someone commented earlier to keep all the config in the gunicorn file.  Haha.

Reply:
:-)

Observe that the [current gunicorn_config.py](https://netbox.readthedocs.io/en/latest/installation/3-http-daemon/#gunicorn-installation) doesn't have an errorlog setting at all - nor is the `--error-log` option given in the [existing supervisord](https://netbox.readthedocs.io/en/latest/installation/3-http-daemon/#supervisord-installation) config.  So you're making an unrelated change here by including it.

You're right that I would not use the env file at all: I consider both the systemd unit file and the gunicorn config file as user-editable config files.  Even if you treat the systemd unit as static application config, there is a systemd 'override' mechanism to allow the user to update individual settings.

But I respect that you want a static systemd unit file with tunables in the env file.  If you're going to do that, *and* you want to set the errorlog option, then I think the consistent approach is to pass `--error-log` using the env variable too.